ecr/pusher.go

```diff
@@ -39,27 +39,32 @@ var (
 // ecrPusher implements the containerd remotes.Pusher interface and can be used
 // to push images to Amazon ECR.
 type ecrPusher struct {
+	// ecrBase provides common ECR specific metadata and services.
 	ecrBase
+	// tracker is used to write out progress to the console.
```

Comment:
nit: not necessarily the console; this is just an interface that can be used for tracking progress

Reply:
Dropped in the latest set of changes - these were mostly my own notes as I paged through code.